e2e/specs/results-viewer.spec.ts

```diff
@@ -0,0 +1,130 @@
+    await page.locator('#manualEntrySerumCreatinineid').fill('10');
+    await page.locator('#manualEntryTotalProteinid').fill('10');
+    await page.locator('#manualEntrySerumGlucosemgdlid').fill('10');
+    await page.locator('#manualEntrySerumGlucosemmolid').fill('10');
+    await page.locator('#manualEntryTotalBilirubinid').fill('10');
+    await page.locator('#manualEntrySerumGlutamicOxaloaceticTransaminaseid').fill('10');
+    await page.locator('#manualEntryAlkalinePhosphastaseid').fill('10');
+    await page.locator('#manualEntrySerumUricAcidid').fill('10');
+    await page.locator('#manualEntryTotalCholesterolid').fill('10');
+    await page.locator('#manualEntryTriglyceridesid').fill('10');
+    await page.locator('#manualEntryAmylaseid').fill('10');
+    await page.locator('#manualEntrySerumCarbonDioxideid').fill('10');
+  });
+
+  await test.step('When I click the `Other` tab', async () => {
+    await page.getByRole('button', { name: 'Other', exact: true }).click();
+  });
+
+  await test.step('And Then I fill the other test results', async () => {
+    await page.locator('#manualEntryCultureandSensitivityUrineid').fill('83%');
+    await page.locator('#manualEntryUrinePregnancyTestid_1').check();
+    await page.locator('#manualEntryUrineProteinDipStickid_0').check();
+    await page.locator('#manualEntryUrineBacteriuriaTestid_1').check();
+    await page.locator('#manualEntryErythrocytesPresenceInUrineSedimentByLightMicroscopyTestid_1').check();
+    await page.locator('#manualEntryLeukocytesPresenceInUrineSedimentByLightMicroscopyid_2').check();
+    await page.locator('#manualEntryEpithelialCastsPresenceInUrineSedimentByLightMicroscopyTestid_2').check();
+    await page.locator('#manualEntryYeastPresenceInUrineSedimentByLightMicroscopyid_1').check();
+    await page.locator('#manualEntrySporePresenceInUrineTestid_3').check();
+    await page.locator('#manualEntryTrichomonasVaginalisPresenceInUrineSedimentByLightMicroscopyid_0').check();
+    await page.locator('#manualEntryCrystalsTypeInUrineSedimentByLightMicroscopyTestid_3').check();
+    await page.locator('#manualEntryStoolCultureBacterialid').fill('no signs');
+    await page.locator('#manualEntryStoolExamid').selectOption('Cestode Infection');
+    await page.locator('#manualEntryStoolMicroscopyWithConcentrationid').fill('no signs');
+    await page.locator('#manualEntryKinyounsStainForCoccidiansid').fill('no signs');
+    await page.locator('#manualEntryStoolFatTestSemiQuantitativeid_3').check();
+    await page.locator('#manualEntryStoolTestforReducingSubstanceid_0').check();
+    await page.locator('#manualEntryFecalOccultBloodTestid_2').check();
+  });
+
+  await test.step('And I save the form', async () => {
+    await page.getByRole('button', { name: /save and close/i }).click();
+  });
+
+  await test.step('Then I should see a success toast notification', async () => {
+    await expect(page.getByText('Laboratory Test Results')).toBeVisible();
+  });
+});
```

Comment:
We should check whether entered information is properly saved and rendered.

Reply:
@jayasanka-sack we are checking for this now. PR was WIP earlier.
Thanks